index.bs

```diff
@@ -127,60 +129,94 @@ div.enum-description > table > thead > tr > th {
 
   <h3 id="autoplay-detection-methods">The Autoplay Detection Methods</h3>
     <pre class="idl">
-      callback HTMLMediaElementConstructor = HTMLMediaElement ();
-      callback AudioContextConstructor = AudioContext ();
+      enum AutoplayPolicyMediaType { "mediaElement", "audioContext" };
 
       [Exposed=Window]
       partial interface Navigator {
-        AutoplayPolicy getAutoplayPolicy(HTMLMediaElementConstructor mediaCtor);
-        AutoplayPolicy getAutoplayPolicy(AudioContextConstructor audioCtor);
+        AutoplayPolicy getAutoplayPolicy(AutoplayPolicyMediaType type);
         AutoplayPolicy getAutoplayPolicy(HTMLMediaElement element);
         AutoplayPolicy getAutoplayPolicy(AudioContext context);
       };
     </pre>
 
-    <h4 id="query-by-a-constructor">Query by a Constructor</h4>
-      The {{getAutoplayPolicy(mediaCtor)}} and {{getAutoplayPolicy(audioCtor)}}
-      methods return the rough status of whether media, which have the same type
-      of the queried constructor and exist in the {{document}} contained in the
-      {{Window}} object associated with the queried {{Navigator}} object, are
-      allowed to autoplay or not. The media terms appearing in the next
-      following if-statement have the same definition as above description.
+    <div class="enum-description">
+      <table class="simple" dfn-for="AutoplayPolicyMediaType" dfn-type="enum-value">
+        <thead>
+          <tr>
+            <th scope="col" colspan="2">
+              Enumeration description
+        <tbody>
+          <tr>
+            <td>
+              <dfn>mediaElement</dfn>
+            <td>
+              It's used to query a status for {{HTMLMediaElement}} and its
+              extensions, such as {{HTMLVideoElement}} and {{HTMLAudioElement}}.
+          <tr>
+            <td>
+              <dfn>audioContext</dfn>
+            <td>
+              It's used to query a status for {{AudioContext}}.
+      </table>
+    </div>
+
+    <h4 id="query-by-a-media-type">Query by a Media Type</h4>
+      The {{getAutoplayPolicy(type)}} methods return the **rough** status of
```

Comment:
Can we be more precise than "rough"? If it's true that the method returns the correct policy at the time the method is called, but the policy may change, can we set some expectation for when the value is stable, eg., in terms of the event loop?

Reply:
The detailed explanation about this are in the following examples and notes, `rough` means it's not 100% correct because the query by an element can return a different result. "Rough" isn't indicating the characteristic of the result it time various, because all of those results returned by different API are having that characteristic. 

But yes, I will add more explanation for this part. Thanks!